### ISSUE TITLE
Is it possible to read out a FT80?

### ISSUE THREAD

**Issue by kami83** [samop/Polar-Flowlink-linux#1]
Hi,

i got a FT80 clock with Flowlink, is it possible to read out the Data?

Cu kami

**Comment by samop** [comment]
No, unfortunately not yet. There has been some effort done in this direction, but it is still far from the completion. Anyway, I am waiting for some time to completely rewrite the code, since it is a mess. It will be easier to adapt the code to the new models.

**Comment by kami83** [comment]
hi,

so what do you mean with new models? Can i help you in anyway to get the data from a ft80?

Is there any other way to connect a ft80 with a linux system?

Cu kami

**Comment by GHPS** [comment]
I'm still interested in using my FT80 with Linux - is there a way to support
the development of a FT80 model?
